src/object_store.jl

```diff
@@ -22,14 +22,9 @@ is blocking until the data is available in the local object store.
 If the task that generated the `ObjectRef` failed with a Julia exception, the
 captured exception will be thrown on `get`.
 """
-get(obj_ref::ObjectRef) = get(obj_ref.oid)
-
-function get(oid::ray_jll.ObjectIDAllocated)
-    local ray_obj
-    while true
-        ray_obj = ray_jll.get(oid, 0)
-        isnull(ray_obj[]) ? sleep(0.1) : break
-    end
+function get(obj_ref::ObjectRef)
+    wait(obj_ref)
+    ray_obj = ray_jll.get(obj_ref.oid, -1)
```

Comment:
Should we set the timeout to zero just in case something weird happens and we don't block forever?
```suggestion
    ray_obj = ray_jll.get(obj_ref.oid, 0)
```

Reply:
I dunno, there _might_ be some kind of weird race condition here that prevents it from actually returning RIGHT away.  I guess we could switch to polling `get` at this point though...or check for a nullptr and throw an error (and if folks encounter it too often, that's a signal for us to fix it...)